### ISSUE TITLE
Support for the upstream keybinding configuration variables

### ISSUE THREAD

**Issue by wshanks** [laloch/xontrib-fzf-widgets#8]
How do you feel about supporting the keybinding variables used by upstream `fzf` as described [here](https://github.com/junegunn/fzf#key-bindings-for-command-line)? Mainly, I am thinking of `FZF_CTRL_T_OPTS` and `FZF_CTRL_R_OPTS`.

For what it's worth, I mainly want this to override the `--reverse` in `fzf_insert_history()` because I am used to the default behavior of the bash keybindings that don't use `--reverse`.

**Comment by laloch** [comment]
Yes, you're right once again. I'm thinking about this change together with replacing all the configuration variables with upper-case versions since I adopted the project.
Maybe it's the time for some serious refactoring and for major version bump.
